{{cookiecutter.project_dashed}}-src/ansible/provision.yaml

```diff
@@ -12,7 +12,15 @@
   tags:
     - web
   roles:
-    - { role: pyapp3, pyapp3_op: provision, tags: pyapp3-role}
+    - role: letsencrypt
+      letsencrypt_op: provision
+      letsencrypt_domains: "{{ [ nginx_vhost_server_name ] + nginx_vhost_server_alias }}"
```

Comment:
Add comment above indicating that order of domains is important and drives the directory name of the certificate installs.